### PR TITLE
[stable/rabbitmq] disable rbac by default

### PR DESCRIPTION
**Pull request by tompizmor**
By default, we should not create RBAC resources unless using the values-production.yaml.

### CODE REVIEW

**Comment by unguiculus** [comment]
Our current RBAC best practices suggest to enable it by default.

**Comment by prydonius** [comment]
Agreed, I don't think we should be disabling it by default unless there's a very good reason.

File: stable/rabbitmq/values.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ image:
   #   - myRegistrKeySecretName
 
 ## does your cluster have rbac enabled? assume yes by default
```
Comment (md-mq) [inline stable/rabbitmq/values.yaml:26]: Maybe update the comment too?

**Comment by tompizmor** [comment]
You're right, it is ok that RBAC is enabled by default. Closing this PR

**Pull request closed without merging**